ratis-client/src/main/java/org/apache/ratis/client/RaftClient.java

```diff
@@ -58,8 +58,8 @@
   /** Get the {@link GroupManagementApi} for the given server. */
   GroupManagementApi getGroupManagementApi(RaftPeerId server);
 
-  /** Get the {@link SnapshotManagementApi}. */
-  SnapshotManagementApi getSnapshotManagementApi();
+  /** Get the {@link SnapshotManagementApi} for the given server. */
```

Comment:
We should support null as below:
```
  /**
   * If server != null, return the {@link SnapshotManagementApi} for the given server.
   * Otherwise, return the {@link SnapshotManagementApi} for the group.
   */
```